src/js/components/MainPanel/mainPanel.jsx

```diff
@@ -19,19 +19,22 @@ import PropTypes from 'prop-types';
 import { useSelector } from 'react-redux';
 import { MainPanelWrapper } from './mainPanel.style';
 import GraphComponent from '../Graphs/graph';
-import { graphJson } from '../Graphs/exampleComponent.mock';
 
 const MainPanel = ({ tabId }) => {
-  const renderedGraph = useSelector(({ pageData }) => pageData[tabId].renderedGraph);
+  const renderedGraphId = useSelector(({ pageData }) => pageData[tabId].renderedGraph);
   const sidebarExpanded = useSelector(({ pageData }) => pageData[tabId].sidebarExpanded);
+  const graphData = useSelector(({ pageData }) => (renderedGraphId
```

Comment:
Please provide tests for:
- `null` - it does not throw an exception
- example graph data - it does not throw an exception